Sources/Common/Util/KeyValueStorage.swift

```diff
@@ -37,25 +35,21 @@ public protocol KeyValueStorage: SharedKeyValueStorage {
  */
 // sourcery: InjectRegister = "KeyValueStorage"
 public class UserDefaultsKeyValueStorage: KeyValueStorage {
-    private var siteId: String?
+    // Set this value before calling any other function in this class, to make sure you get/set values in a sandboxed key/value store.
+    // This solution is fragile because if you forget to set the siteid, you will get/set from the global data store.
+    // This temporary solution will be replaced with a better solution.
+    var siteId: String?
```

Comment:
[The next PR](https://github.com/customerio/customerio-ios/pull/547) in the stack replaces this fragile implementation with something more reliable.